concepts/comparisons/introduction.md

```diff
@@ -1,23 +1,65 @@
-This is also called `short-circuit` evaluation which means the execution is stopped if the truth value of the expression has already been determined.
-Note that the evaluation of expression takes place from left to right.
-In python, short circuiting is supported by various boolean operators, functions and, in this case, comparison chaining.
 
-## Comparison of different data types
+| Operator | Operation                | Description                                                    |
+| -------- | ------------------------ | -------------------------------------------------------------- |
+| `>`      | "greater than"             | `a > b` is `True` if `a` is **strictly** greater in value than `b` |
+| `<`      | "less than"                | `a < b` is `True` if `a` is **strictly** less in value than `b`    |
+| `==`     | "equal to"                 | `a == b` is `True` if `a` is **strictly** equal to `b` in value    |
+| `>=`     | "greater than or equal to" | `a >= b` is `True` if `a > b` OR `a == b` in value             |
+| `<=`     | "less than or equal to"    | `a <= b` is `True` if `a < b` or `a == b` in value             |
+| `!=`     | "not equal to"             | `a != b` is `True` if `a == b` is `False`                      |
+| `is`     | "identity"                 | `a is b` is `True` if **_and only if_** `a` and `b` are the same _object_  |
+| `is not` | "negated identity"                 | `a is not b` is `True` if `a` and `b` are **not** the same _object_ |
+| `in`     | "containment test"         | `a in b` is `True` if `a` is member, subset, or element of `b`               |
+| `not in` | "negated containment test" | `a not in b` is `True` if `a` is not a member, subset, or element of `b`     |
 
-Since everything in Python is an `object`, things can get interesting when objects of different types are compared.
-For example, the `str` value of a number is considered completely different from the `integer` or `floating-point` value.
-However, an `integer` **can** be considered equal to a `float`, as they are both numeric types that Python can implicitly convert to compare.
-For other numeric types, comparison operators are defined where they "make sense", but throw a `TypeError` if the underlying objects cannot be converted for comparison.
-For more information on the rules that python uses for numeric conversion, see [arithmetic conversions][arithmetic conversions] in the Python documentation.
 
-[comparisons]: https://docs.python.org/3/library/stdtypes.html?
-[arithmetic conversions]: https://docs.python.org/3/reference/expressions.html?highlight=number%20conversion#arithmetic-conversions
+They all have the same priority (_which is higher than that of [Boolean operations][boolean operations], but lower than that of arithmetic or bitwise operations_).
+
+

```

Comment:
Two blank spaces which is usually ignored in HTML rendering, and not really helpful in the plain text viewing.  This happens in other places as well.  Single blank line between parenthesis is likely intended.

Reply:
I am not sure I follow here.  Which two spaces are we talking about?

Reply:
Two blank vertical spaces (new lines) is what I really mean here.